src/main/java/uk/gov/justice/hmpps/casenotes/controllers/CaseNoteController.java

```diff
@@ -77,7 +80,10 @@ public CaseNote getCaseNote(
     public CaseNote createCaseNote(
             @ApiParam(value = "Offender Identifier", required = true, example = "A1234AA") @PathVariable("offenderIdentifier") final String offenderIdentifier,
             @RequestBody @NotNull final NewCaseNote newCaseNote) {
-        return caseNoteService.createCaseNote(offenderIdentifier, newCaseNote);
+        final var caseNoteCreated = caseNoteService.createCaseNote(offenderIdentifier, newCaseNote);
```

Comment:
Decided to add into the controller as the transaction has definitely completed by this point.